index.html

```diff
@@ -41,14 +41,13 @@
     </style>
   </head>
 
-  <body>
+  <body class="body body__active">
```

Comment:
sorry about another classname related comment, it's minor but - `active` is a modifier so it should be `body--active`. Double underscore is for elements (so often children)